src/components/utils/unit_tests/onfidoApi.test.js

```diff
@@ -0,0 +1,80 @@
+import { sendMultiframeSelfie } from "../onfidoApi"
+
+const jwtToken = 'my_token'
+
+const snapshotData = {
+  blob: new Blob,
+  filename: 'applicant_selfie.jpg'
+}
+
+const selfieData = {
+  blob: new Blob,
+  filename: 'applicant_selfie.jpg',
+  sdkMetadata: {}
+}
+
+const url = "https://test.url.com"
+
+const createMockXHR = () => ({
+    open: jest.fn(),
+    onload: jest.fn(),
+    send: jest.fn(),
+    setRequestHeader: jest.fn(),
+    status: 201,
+    response: JSON.stringify({"payload": "success"})
+})
+
+const oldXMLHttpRequest = window.XMLHttpRequest;
+let mockXHR = null;
+
+beforeEach(() => {
+  mockXHR = createMockXHR();
+  window.XMLHttpRequest = jest.fn(() => mockXHR);
+});
+
+afterEach(() => {
+  window.XMLHttpRequest = oldXMLHttpRequest;
+});
+
+const onSuccess = (res) => {
+  expect(res).toMatchObject({"payload": "success"})
+}
+const onError = (res) => {
+  expect(res).toMatchObject({status: 401, response: {"error": "unauthorized"} })
+}
+
+describe('sendMultiframeSelfie', () => {
+  describe('with valid data', () => {
+    it('should send two XHR requests', async() => {
+      sendMultiframeSelfie(snapshotData, selfieData, jwtToken, url, onSuccess, onError)
+      mockXHR.onload(); // First XHR has been mocked
+      await Promise.resolve();
+      mockXHR.onload(); // Second XHR has been mocked
+      expect.assertions(2); // There are two assertion in this test case because onSuccess is calling `expect` too
+      expect(mockXHR.send.mock.calls.length).toBe(2) // Two request send() methods have been called
+    })
+  })
+
+  // TODO: Re-enable this tests once the backend endpoint is in production
```

Comment:
We are going to have the backend in production ready after this platform release, aren't we?

Reply:
They will let us know by Thursday